packages/syntax-core/src/RichSelect/RichSelectRadioButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef, type ReactElement } from "react";
+import RichSelectItem, { type RichSelectItemProps } from "./RichSelectItem";
+import RadioButton from "../RadioButton/RadioButton";
+
+const NOOP = () => undefined;
+
+type RichSelectRadioButtonProps = RichSelectItemProps & {
+  name: string;
+  size?: "sm" | "md";
+};
+
+export default forwardRef<HTMLDivElement, RichSelectRadioButtonProps>(
+  function RichSelectRadioButton(props, ref): ReactElement {
+    const { size = "sm", name, ...otherProps } = props;
```

Comment:
remove default "sm" for now?

Reply:
removed the default.  I'd like to make the `size` prop for the `RichSelectList/Box` become the context for the size of all the buttons and input elements contained within as a followup when the need/time arises